docs/developers/faqs.md

```diff
@@ -0,0 +1,377 @@
+---
+sidebar_label: Frequently Asked Questions (FAQs)
```

Comment:
Why is this file being added?

Reply:
i don't know. i think it may have come from this branch being made before the change from tuesday.

i resolved merge conflicts in browser here: https://github.com/celestiaorg/docs/pull/1051/commits/a167d2dc36b682dcc7f7832168ce95e3907e1849

and then removed quotes from evan's suggestion: https://github.com/celestiaorg/docs/pull/1051/commits/622c654a19540ae76d0ee7d1bf967a3a8c2b9dcb